docs/data/date-pickers/calendar-systems/AdapterHijri.tsx

```diff
@@ -41,16 +40,18 @@ function ButtonDateTimeField(props: DateTimePickerFieldProps) {
   } = forwardedProps;
 
   const pickerContext = usePickerContext();
-
-  const parsedFormat = useParsedFormat(internalProps);
+  const parsedFormat = useParsedFormat();
   const { hasValidationError } = useValidation({
```

Comment:
We would now be able to make `value` and `timezone` optional and read it from the context by default. Same with the validation props in the future.

But I would like to take the time to think this API correctly before doing anything.